packages/oui-skeleton/README.md

```diff
@@ -0,0 +1,20 @@
+# oui-skeleton
+
+<component-status cx-design="complete" ux="beta"></component-status>
+
+oui-skeleton is a package which provides styles for the skeleton component.
+
+## Installation
+
+```less
+  @import 'oui-skeleton/skeleton';
+```
+
+## Usage
+
+
+```html:preview
+<div class="oui-skeleton">
+    <span class="oui-skeleton__loader" style="width: 30%"></span>
```

Comment:
No inline style in our preview, that could be copy/pasted.
We have classnames helpers for that. Check for `oui-doc-preview-only` and `oui-doc-preview-only-keep-children`.